### ISSUE TITLE
UI/UX design

### ISSUE THREAD

**Issue by SarthakKeshari** [sairish2001/makesmatheasy#641]
## Proposal

Sir @sairish2001,
      I wish to prepare a UI/UX upgrade for this website. I would be using Adobe XD for achieving this goal. Plan is not the give it a totally different outlook but to make proper amendments in UI at place where its needed. I will keep in mind the consious measures to take so that the whole website look need not be disturbed.
      Sir, if you want a complete new outlook I am eagar enough to do the same. Once I get you approval over the proposal I will start working on the same, then once everything is in place I will document it and ask for your advice over the corrections required.

Thankyou.

**Comment by sairish2001** [comment]
If you feel you can create a good wireframe to give it a different look, then feel free to do the same.

Also keep it in mind, it's a webapp, more focuses on creating a Mobile App like look. So design accordingly.

**Comment by sairish2001** [comment]
Go ahead
3 April,2021 (11:59 PM IST)

**Comment by SarthakKeshari** [comment]
Sure sir... I will keep these points in mind

**Comment by SarthakKeshari** [comment]
Sir @sairish2001, 
Since I am designing for destop and mobile view both, I request you to increase the deadline till 8th April.

**Comment by sairish2001** [comment]
Deadline extended 9 April,2021

**Comment by sairish2001** [comment]
if any of either is ready, do share the screenshot

**Comment by SarthakKeshari** [comment]
I am prototyping the mobile version and will send a working video or adobe link atmost by tomorrow evening

**Comment by sairish2001** [comment]
@SarthakKeshari  Any update?

**Comment by SarthakKeshari** [comment]
No, Due to unavoidable circumstances I am unable to complete this issue.
You may close this issue for me

**Comment by sairish2001** [comment]
Go ahead
@anjali1102
